source/client/main.qc

```diff
@@ -1166,7 +1167,11 @@ noref void() CSQC_Parse_Event =
 			screenflash_starttime = time;
 			break;
 		case CSQC_EVENT_RUMBLE:
-			gp_rumble(last_input_deviceid, readshort(), readshort(), readshort());
+			float high_frequency = readshort();
```

Comment:
This is backwards, low frequency is networked to the client first. Then high frequency.

Reply:
Nice catch. Will fix it.